### PR TITLE
EDGINREACH-7

### PR DESCRIPTION
**Pull request by AnShapovalov**
## Pre-Merge Checklist:
 Before merging this PR, please go through the following list and take appropriate actions.

 - Does this PR meet or exceed the expected quality standards?
   - [x] Code coverage on new code is 80% or greater
   - [x] Duplications on new code is 3% or less
   - [ ] There are no major code smells or security issues
 - Does this introduce breaking changes?
   - [ ] Were any API paths or methods changed, added or removed?
   - [ ] Were there any schema changes?
   - [ ] Did any of the interface versions change?
   - [ ] Were permissions changed, added, or removed?
   - [ ] Are there new interface dependencies?
   - [ ] There are no breaking changes in this PR.

 If there are breaking changes, please **STOP** and consider the following:

 - What other modules will these changes impact?
 - Do JIRAs exist to update the impacted modules?
   - [ ] If not, please create them
   - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
   - [ ] Do they have all they appropriate links to blocked/related issues?
 - Are the JIRAs under active development?
   - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
 - Do PRs exist for these changes?
   - [ ] If so, have they been approved?

 Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

 While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -25,44 +24,16 @@
...
-    <!-- Api definition files -->
-    <i2nreach.yaml.file>
-      ${project.basedir}/src/main/resources/swagger.api/i2nreach.yaml
-    </i2nreach.yaml.file>
-
+    <openapi-generator.version>4.3.1</openapi-generator.version>
+    <openapi.input.file>${project.basedir}/src/main/resources/swagger.api/inn-reach.yaml</openapi.input.file>
+    <mapstruct.version>1.3.1.Final</mapstruct.version>
+    <json.version>20200518</json.version>
+    <wiremock.version>2.27.2</wiremock.version>
```
Comment (oleksandr-yatsenko1) [inline pom.xml:31]: Do we need to decrease version of wiremock dependency ?

File: pom.xml [hunk 2]
```diff
@@ -264,28 +212,27 @@
...
               <goal>generate</goal>
             </goals>
             <configuration>
-              <inputSpec>${i2nreach.yaml.file}</inputSpec>
+              <inputSpec>${openapi.input.file}</inputSpec>
               <output>${project.build.directory}/generated-sources</output>
               <generatorName>spring</generatorName>
-              <modelPackage>${project.groupId}.edge.i2nreach.domain.dto</modelPackage>
-              <apiPackage>${project.groupId}.edge.i2nreach.rest.resource</apiPackage>
+              <modelPackage>${project.groupId}.edge.dto</modelPackage>
```
Comment (oleksandr-yatsenko1) [inline pom.xml:230]: Looking on another spring-related modules there are used naming convention like:
``
*.<module-semantic-related-name>.domain.dto
*.<module-semantic-related-name>.rest.resource
``
for example. Does it make sense to use here similar approach? If so, some kind of `'*.ir.domain.dto"` and `"*.ir.rest.resource"` or somethoing similar could be used here. How do you think?

File: pom.xml [hunk 3]
```diff
@@ -264,28 +212,27 @@
...
+              <apiPackage>${project.groupId}.edge.rest.resource</apiPackage>
               <generateApis>true</generateApis>
               <generateApiTests>true</generateApiTests>
               <generateApiDocumentation>true</generateApiDocumentation>
               <generateModels>true</generateModels>
```
Comment (oleksandr-yatsenko1) [inline pom.xml:235]: We states here about generating models but removed list of models to generate. Can we skip their listing here?

File: descriptors/ModuleDescriptor-template.json [hunk 4]
```diff
@@ -15,6 +15,16 @@
           "pathPattern": "/_/tenant/{id}"
         }
       ]
+    },
+    {
+      "id": "inn-reach",
+      "version": "2.0",
```
[the remark this answers is not in the document]
Comment (AnShapovalov) [reply descriptors/ModuleDescriptor-template.json:21]: We are implementing specified INN-Reach API 2.0 in this module. That's why here API version is 2.0. If you have any idea how this should be implemented by else way - welcome.

**Pull request merged**